lib/view_component/engine.rb

```diff
@@ -42,21 +43,35 @@ class Engine < Rails::Engine # :nodoc:
       end
     end
 
-    initializer "view_component.monkey_patch_render" do
+    initializer "view_component.monkey_patch_render" do |app|
+      next if Rails.version.to_f >= 6.1 || !app.config.view_component.render_monkey_patch_enabled
```

Comment:
```suggestion
      return if Rails.version.to_f >= 6.1 || !app.config.view_component.render_monkey_patch_enabled
```

Reply:
I think this can't be changed to `return` because we are not inside a method (so it'd result in a `LocalJumpError` exception.